packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -397,6 +428,28 @@ class MessageList extends Widget<Properties> {
     this.setAria(aria);
   }
 
+  _setIsReachedBottom(): void {
+    const scrollTop = this._scrollView.scrollTop();
+    const clientHeight = this._scrollView.clientHeight();
+    const scrollHeight = this._scrollView.scrollHeight();
+
+    const isReachedBottom = Math.floor(scrollHeight) === Math.floor(clientHeight + scrollTop);
```

Comment:
It might not work properly with scaling cases. I would prefer to believe more on the solution used in the List component: https://github.com/marker-dao/DevExtreme/blob/1d5739865c023b6c7a90cfcce7a2845805f29860/packages/devextreme/js/__internal/ui/list/m_list.base.ts#L563-L564

Reply:
Hm, okay. Let's use it. I also created a task to include this code in scrollable, since this is the second case where we need to know this.

Reply:
[91ca30ce4b31045a71987cb15c53c1eddde9c45c]